docs/serving/istio-authorization.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Enabling requests to Knative services when additional authorization policies are enabled"
+weight: 25
+type: "docs"
+---
+
+Knative Serving system pods, such as the activator and autoscaler components, require access to your deployed Knative services.
+If you have configured additional security features, such as Istio's authorization policy, you must enable access to your Knative service for these system pods.
+
+## Before you begin
+
+You must meet the following prerequisites to use Istio AuthorizationPolicy:
+
+- [Enabling istio sidecar injection](https://istio.io/latest/docs/setup/additional-setup/sidecar-injection/).
+- Using net-istio for your Knative Ingress.
```

Comment:
What does this mean? Sorry, I'm not familiar with working directly on Knative.

Reply:
No problem. Knative can choose the network layer Istio,Kourier, Ambassador and so on. You can refer to:

https://knative.dev/docs/install/any-kubernetes-cluster/#installing-the-serving-component
> 3. Pick a networking layer (alphabetical):

I added the link for the doc.